Assets/Scripts/Gameplay/GameplayObjects/FloorSwitch.cs

```diff
@@ -34,14 +34,14 @@ void Awake()
 
         public override void OnNetworkSpawn()
         {
-            if (!IsServer)
+            if (IsServer)
             {
                 enabled = false;
```

Comment:
I think this should be enabled on the server. While testing I found the door to not be opening

Reply:
Changing this solves the issue, but I found another bug. The door opening/closing message appears twice in the message feed for the host each time a change happens. On clients though everything seems fine

Reply:
nvm found the cause for the duplicated messages: https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/732/files#r976925850

Reply:
resolved